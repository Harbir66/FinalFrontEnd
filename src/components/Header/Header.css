.header {
  display: flex;
  align-items: center;
  background-color: #FFFFFF;
  padding-left: 5vw;
  width: 75vw;
  height: 7vh;
  color: #4F4F4F;
  font-weight: bold;
  font-size: 2vh;
}
