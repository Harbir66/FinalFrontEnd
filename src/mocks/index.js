export const mockCollection = [
  {
    id: 1,
    name: 'collection1',
    fields: 3,
    values: 3,
    allFields: ['field1', 'field2', 'field3'],
  },
  {
    id: 2,
    name: 'collection2',
    fields: 3,
    values: 3,
    allFields: ['field1', 'field2', 'field3'],
  },
];

export const mockHeading = 'Heading';

export const mockFields = ['field1', 'field2', 'field3'];
