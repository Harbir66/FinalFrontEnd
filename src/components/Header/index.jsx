import React from 'react';
import './Header.css';
import PropTypes from 'prop-types';

function Header({ heading }) {
  return <header className="header">{heading}</header>;
}

export default Header;

Header.propTypes = {
  heading: PropTypes.string.isRequired,
};
