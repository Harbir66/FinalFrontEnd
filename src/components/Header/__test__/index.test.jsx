import { render } from '@testing-library/react';
import React from 'react';
import Header from '../index';
import { mockHeading } from '../../../mocks';

describe('Header', () => {
  it('should render', () => {
    const { asFragment } = render(<Header heading={mockHeading} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
