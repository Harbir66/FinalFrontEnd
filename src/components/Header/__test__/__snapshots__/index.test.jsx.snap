// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render 1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    Heading
  </header>
</DocumentFragment>
`;
